### PR TITLE
Htv3hc/feat/trigger other workflow

### PR DESCRIPTION
**Pull request by huavanthong**
Hi Thomas,

I would like to update the code to resolve issue on https://github.com/test-fullautomation/RobotFramework_AIO/issues/137
* **Issues 1:** to trigger a pipeline when a new-pull-request, or updated-pull-request coming. We can use keyword **opened** and **synchronize**. 

**Note:** If we close a pull request, it also trigger pipeline as same as the process before.
```yaml
pull_request:
    types:
      - closed
      - opened
      - synchronize
```
* **Issues 2:** To trigger pipeline when a pull request come from other repositories.

The workflow of Robotframework_AIO following info below: 
1. I created a job named by **trigger-aio** to listen event from other repositories.
2.  That job will be worked following 2 use case:
* **Case 1:**  A new-pull-request, or updated-pull-request at this repos => Pipeline will be triggered without running **trigger-aio** jobs.
* **Case 2:**  A new-pul request, or updated-pull-request by other repostories => Pipliene will be triggered with running **trigger-aio** jobs. 
===>  After **trigger-aio** it will get info from other repository such as REPOSITORY NAME and BRANCH NAME. It will replace the value in **repositories.conf** to make this RobotFramework_AIO clone other repositories with the target branch of pull request .
===> Then it will combine all code for building.

Could you help me review my pull-request? 

Thank you and best regards,
Thong Hua

### CODE REVIEW

File: cloneall [hunk 1]
```diff
@@ -133,6 +133,14 @@ if [ -f $config_file ]; then
 else
 	errormsg "Repo configuration '$config_file' is not existing"
 fi
+
+# Receive event from other repos and update repository config file
+if [ -n "$REPOSITORY"  ] && [ -n "$PULL_REQUEST_BRANCH" ]; then 
+
+	sed -i "s/$REPOSITORY=\(.*\)/${REPOSITORY}=$PULL_REQUEST_BRANCH/" "$CONFIG_FILE"
```
Comment (ngoan1608) [inline cloneall]: Variable for location of repo configuration should be `$config_file` (all lower-case characters).

**Review by ngoan1608 — CHANGES_REQUESTED** [review]

File: cloneall (hunk 1 above)
Comment (huavanthong) [reply cloneall]: Thank you for your info. I have updated it.

**Comment by test-fullautomation** [comment]
Hi Thong,
thank you for your update.
I merge this directly after we released our 0.9.0 version.
This has highest priority as we are unfortunately already overdue.
Thank you,
 Thomas

**Review by test-fullautomation — APPROVED** [review]
Hi Thong,
looks good to me!
Thank you,
 Thomas

**Pull request merged**